iot_config/migrations/20241203190910_solana_organizations.sql

```diff
@@ -0,0 +1,39 @@
+-- Migration here solely for testing purposes
+-- An instance of account-postgres-sink
+-- Will alter this table depending on an on-chain struct
+CREATE TABLE IF NOT EXISTS solana_organizations (
+    address TEXT PRIMARY KEY,
+    net_id TEXT NOT NULL,
+    authority TEXT NOT NULL,
+    oui BIGINT NOT NULL,
+    escrow_key TEXT NOT NULL,
```

Comment:
Is this the key where the funds are coming from? aka why not keeping it `payer_key`?